src/pan3d/ui/layouts.py

```diff
@@ -0,0 +1,143 @@
+"""Standard layout templates for Pan3D explorers."""
+
+from pan3d.ui.vtk_view import Pan3DView
+from pan3d.utils.common import ControlPanel
+from pan3d.widgets.error_alert import ErrorAlert
+from pan3d.widgets.save_dataset_dialog import SaveDatasetDialog
+from pan3d.widgets.scalar_bar import ScalarBar
+from pan3d.widgets.time_navigation import TimeNavigation
+from trame.ui.vuetify3 import VAppLayout
+from trame.widgets import vuetify3 as v3
+
+
+class StandardExplorerLayout:
```

Comment:
What don't you use a base class VAppLayout so it is truly a layout where you can define slots.